src/wallet/wallet.cpp

```diff
@@ -1661,14 +1661,22 @@ bool CWallet::DummySignTx(CMutableTransaction &txNew, const std::vector<CTxOut>
     return true;
 }
 
-bool CWallet::ImportScripts(const std::set<CScript> scripts)
+bool CWallet::ImportScripts(const std::set<CScript> scripts, int64_t time)
 {
     WalletBatch batch(*database);
     for (const auto& entry : scripts) {
         if (!HaveCScript(CScriptID(entry)) && !AddCScriptWithDB(batch, entry)) {
             return false;
         }
+
+        if (time >= 0) {
```

Comment:
In commit "Optionally allow ImportScripts to set script creation timestamp" (92863d7de8cf0d7432bc47f36bf1aa4de257c548)

This is inconsistent with existing code which uses 0 to indicate an unknown time and 1 to indicate the earliest timestamp. 

Would suggesting replacing `(time >= 0)` to `(time > 0)` here and below and changing default to 0 instead of -1. (Or dropping the default entirely).

Reply:
Done